nexus_keycode/test/protocols/test_full.py

```diff
@@ -265,7 +266,23 @@ def test_display_payg_id__verify_output__ok(self):
 
         self.assertEqual(
             msg.header,
-            "{:01d}".format(protocol.FullMessageType.FACTORY_DISPLAY_PAYG_ID.value),
+            u"{:01d}".format(protocol.FullMessageType.FACTORY_DISPLAY_PAYG_ID.value),
         )
         self.assertEqual(msg.body, "")
         self.assertEqual("*634 776 5#", keycode)
+
+    def test_passthrough_command__channel_command__expected(self):
+        # Send an arbitrary PAYG_UART_PASSTHROUGH message
+        msg = protocol.FactoryFullMessage.passthrough_command(
+            protocol.PassthroughApplicationId.TO_PAYG_UART_PASSTHROUGH,
+            passthrough_digits="9238284782879",
```

Comment:
Can we add a unit test with one fewer digit here? It should raise due to the total length being 14.